css/style-gallery.css

```diff
@@ -0,0 +1,127 @@
+
+/*Gallery*/
+.gallery { 
+  max-width: 1500px;
+  min-height: 1436px;
+  background-color: #ffe92f;
+  margin: 0 auto;
+}
+
+.gallery-header { 
+  position: relative;
+  top: 100px;
+}
+
+.gallery-header__title { 
+  font-family: "Raleway", sans-serif;
+  font-size: 90px;
+  font-weight: bold;;
+  line-height: 0.3;
+  text-align: center;
+  color: #262626;
+}
+
+.gallery-header__text { 
+  width: 577px;
+  height: 70px;
+  font-family: "Raleway", sans-serif;
+  font-size: 17px;
+  font-weight: 300;
+  line-height: 1.59;
+  text-align: center;
+  color: #262626;
+  margin: 0 auto;
+  position: relative;
+  top: 60px;
+}
+
+/*gallery-section*/
+.gallery-section { 
+  width: 1170px;
+  margin: 0 auto;
+}
+
+.gallery-section-block-first { 
+  display: flex;
+  justify-content: center;
+  flex-wrap: wrap;
+  position: relative;
+  top: 250px;
+}
+
+.gallery-section-block-first__img-first { 
+  width: 770px;
+  height: 550px;
+  position: relative;
+  left: 80px;
+}
+
+.gallery-section-block-first__img-first--img { 
+  width: 80%;
+}
+
+.gallery-section-block-first__img-second { 
+  width: 370px;
+  height: 260px;
+  position: relative;
+  right: 35px;
+}
+
+.gallery-section-block-first__img-second--img { 
+  width: 80%;
+}
+
+.gallery-section-block-first__img-three { 
+  width: 370px;
```

Comment:
Also, since every section has common `width` and `height` would be nice to have those rules in separate class/selector